### ISSUE TITLE
Newlines in Blog Posts

### ISSUE THREAD

**Issue by lane-eden** [dekkerglen/CubeCobra#114]
Currently, in order to have multiple paragraphs in a blog post you must have an empty line between the paragraphs and put a space on that line. New lines should work properly.

**Comment by emmett9001** [comment]
Can you provide detailed replication instructions for this issue? This blog post was created with only a single newline, and it appears to work properly.

![Screen Shot 2019-08-30 at 4 52 58 PM](https://user-images.githubusercontent.com/723615/64056244-acdfad00-cb46-11e9-8fff-fc9524ec90e2.png)

**Comment by jwallra** [comment]
Here's a link to my blog post which has the same problem.  I copied-and-pasted from CT, and added new lines when they didn't show up.  They went away again when I saved.  It looks like the editor is just inserting tons of empty divs instead:

![image](https://user-images.githubusercontent.com/13154238/65392597-6c311900-dd2b-11e9-8154-e43294e774e9.png)

Here's a link to the blog posts: https://cubecobra.com/cube/blog/1oo
